tests/components/deconz/test_switch.py

```diff
@@ -98,24 +95,15 @@ async def test_switches(hass):
         "state": {"on": False},
     }
     gateway.api.event_handler(state_changed_event)
-    state_changed_event = {
-        "t": "event",
-        "e": "changed",
-        "r": "lights",
-        "id": "3",
-        "state": {"alert": None},
-    }
-    gateway.api.event_handler(state_changed_event)
-    await hass.async_block_till_done()
 
-    on_off_switch = hass.states.get("switch.on_off_switch")
-    assert on_off_switch.state == "off"
+    assert hass.states.get("switch.on_off_switch").state == "off"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6271139)